eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/ConfigResolverParameterPass.php

```diff
@@ -159,9 +159,10 @@ private function injectFakeService( ContainerBuilder $container, $configResolver
     {
         $paramConverter = new Definition( 'stdClass', $configResolverArgs );
         $paramConverter
-            ->setFactoryService( 'ezpublish.config.resolver' )
-            ->setFactoryMethod( 'getParameter' )
-            ->setSynchronized( true );
+            ->setFactory( array( new Reference( 'ezpublish.config.resolver' ), 'getParameter' ) )
+            // @deprecated Synchronized services are deprecated in Symfony 2.7 in favour of RequestStack
+            // see: https://github.com/symfony/symfony/pull/13289
+            ->setSynchronized( true, false );
```

Comment:
This will be a problem for us, for sure... I don't see how to workaround this future removal.
Ping @fabpot maybe?